src/kubeChannel.ts

```diff
@@ -0,0 +1,20 @@
+import * as vscode from "vscode";
+
+export interface IKubeChannel {
+    showOutput(message: any, title?: string);
+}
+
+class KubeChannel implements IKubeChannel {
+    private readonly channel: vscode.OutputChannel = vscode.window.createOutputChannel("Kubernetes");
+
+    showOutput(message: any, title?: string) {
+        if (title) {
+            const HIGHLIGHTING_TITLE = `[${title} ${(new Date()).toISOString().replace(/z|t/gi, ' ').trim()}]`;

```

Comment:
Local variables (even when constant) should be `camelCase` not `SHOUTY_CAPS`.  I also think it would be useful if you pulled out the `(new Date()).toISOString().replace(/z|t/gi, ' ').trim()` expression into a named variable - it takes a bit of eyeballing to figure out what it's doing and it would be good to save the reader that.

Reply:
Nice suggestion. thanks.